app/app/static/custom/ourscript.js

```diff
@@ -0,0 +1,117 @@
+var timers = [];
+var timerIntervals = [];
+
+function showAddToQueue() {
+	$('#addToQueue').css('display', 'block');
+}
+
+function hideAddToQueue() {
+	document.getElementById('addToQueueForm').reset();
+	$('#addToQueue').css('display', 'none');
+}
+
+$('#addToQueueForm').submit(function (e) {
+	e.preventDefault();
+
+	let name = document.getElementById('studentName').value;
+	let id = document.getElementById('studentNumber').value;
+	let unit = document.getElementById('unitCode').value;
+	let team = document.getElementById('team').value;
+	let enquiry = document.getElementById('enquiryType').value;
+
+	if (team == 'STUDYSmarter') {
+		var table = $('#SSQueueTable tbody');
+	} else {
+		var table = $('#libQueueTable tbody');
+	}
```

Comment:
this can be replaced to
```js
let table = $(`#${team == 'STUDYSmarter' ? 'SS' : 'lib'}QueueTable tbody`);
```

making use of "ternary" and string templating